packages/orm/src/config.ts

```diff
@@ -28,7 +28,7 @@ export type EntityHook =
   | "afterValidation"
   | "beforeCommit"
   | "afterCommit";
-type HookFn<T extends Entity, C> = (entity: T, ctx: C) => MaybePromise<void>;
+type HookFn<T extends Entity, C> = (entity: T, ctx: C) => MaybePromise<unknown>;
```

Comment:
suggestion: `MaybePromise<any>` should let you avoid needing the explicit return